yarn-project/simulator/src/client/index.ts

```diff
@@ -1,4 +1,5 @@
 export { AcirSimulator } from './simulator.js';
+export { ViewDataOracle } from './view_data_oracle.js';
```

Comment:
Extra import?

Reply:
I import this in the txe, so that I don't have to re-implement the logic there.